packages/aragon-cli/test/utils.js

```diff
@@ -91,3 +95,16 @@ export async function assertIpfsIsInstalled() {
     throw Error(`IPFS is not installed: ${e.message}`)
   }
 }
+
+export function matchAddressAtLineContaining(str, query) {
+  function matchAddress(str) {
```

Comment:
Found this confusing first time I read it, I assumed these functions are declared here because they are being returned. Perhaps we should avoid nested function declarations?